_inc/lib/core-api/wpcom-endpoints/object-id-range.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Automattic\Jetpack\Sync\Modules;
+
+class WPCOM_REST_API_V2_Endpoint_Object_Id_Range {
+	public function __construct() {
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	public function register_routes() {
+		register_rest_route( 'wpcom/v2', '/object-id-range', array(
+			array(
+				'methods'     => WP_REST_Server::READABLE,
+				'callback'    => array( $this, 'get_range' ),
+				'description' => 'An endpoint to retrieve the minimum and maximum ID from the database table corresponding to the given object type.',
+				'args'        => array(
+					'module_name' => array(
+						'type'    => 'string',
+						'default' => 'posts',
+						'enum'    => array(
+							'posts',
+							'comments',
+							'users',
```

Comment:
Hyper nit: should these be in alphabetical order?

Reply:
Consider it done, my friend.